src/hooks/Rule/topology/bridge/useBridgeTypeValue.ts

```diff
@@ -0,0 +1,51 @@
+import { getLabelFromValueInOptionList } from '@/common/tools'
+import { BridgeStatus, BridgeType } from '@/types/enum'
+import { MQTTBridgeDirection } from '@/types/enum'
+import { useI18n } from 'vue-i18n'
+
+export default () => {
+  const bridgeTypeList = [
+    { value: BridgeType.HTTP, label: 'HTTP' },
+    { value: BridgeType.MQTT, label: 'MQTT' },
+  ]
+
+  const getBridgeLabelByTypeValue = (typeValue: BridgeType) => {
+    return getLabelFromValueInOptionList(typeValue, bridgeTypeList)
+  }
```

Comment:
```js
const getBridgeLabelByTypeValue = (typeValue: BridgeType) =>
	getLabelFromValueInOptionList(typeValue, bridgeTypeList)
```

Reply:
I'm trying to avoid unsightly line breaks under formatting rules, so can I write like this?

Reply:
Not a suggestion that needs attention, nor is it important..

Reply:
YES...

My advice is to not write a carriage return when there is only one line.

But it's not important

Reply:
got it